### ISSUE TITLE
Oci extensions fail to install

### ISSUE THREAD

**Issue by greg0ire** [shivammathur/setup-php#312]
**Describe the bug**

Both `oci8` and `pdo_oci` fail to install with the following messages:

```
PHP Warning:  PHP Startup: Unable to load dynamic library 'pdo_oci.so' (tried: /usr/lib/php/20190902/pdo_oci.so (/usr/lib/php/20190902/pdo_oci.so: cannot open shared object file: No such file or directory), /usr/lib/php/20190902/pdo_oci.so.so (/usr/lib/php/20190902/pdo_oci.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
```

```
PHP Warning:  PHP Startup: Unable to load dynamic library 'oci8.so' (tried: /usr/lib/php/20190902/oci8.so (/usr/lib/php/20190902/oci8.so: cannot open shared object file: No such file or directory), /usr/lib/php/20190902/oci8.so.so (/usr/lib/php/20190902/oci8.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
```

**Version**
- [x] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.
- [x] `v2`
- [ ] `v1`

**Runners**

- [x] GitHub Hosted
- [ ] Self Hosted

**Operating systems**
`ubuntu-20.04`

**PHP versions**
`7.4.11`

**To Reproduce**
```yaml
  phpunit-oci8:
    name: "PHPUnit on OCI8"
    runs-on: "ubuntu-20.04"

    strategy:
      matrix:
        php-version:
          - "7.4"

    services:
      oracle:
        image: "wnameless/oracle-xe-11g-r2"
        ports:
          - "1521:1521"

    steps:
      - name: "Checkout"
        uses: "actions/checkout@v2"
        with:
          fetch-depth: 2

      - name: "Install PHP"
        uses: "shivammathur/setup-php@v2"
        with:
          php-version: "${{ matrix.php-version }}"
          extensions: "oci8"
          coverage: "pcov"
          ini-values: "zend.assertions=1"

      - name: "Cache dependencies installed with composer"
        uses: "actions/cache@v2"
        with:
          path: "~/.composer/cache"
          key: "php-${{ matrix.php-version }}-composer-locked-${{ hashFiles('composer.lock') }}"
          restore-keys: "php-${{ matrix.php-version }}-composer-locked-"

      - name: "Install dependencies with composer"
        run: "composer install --no-interaction --no-progress --no-suggest"

      - name: "Run PHPUnit"
        run: "vendor/bin/phpunit -c ci/github/phpunit/oci8.xml --coverage-clover=coverage.xml"

      - name: "Upload coverage file"
        uses: "actions/upload-artifact@v2"
        with:
          name: "${{ github.job }}-${{ matrix.php-version }}.coverage"
          path: "coverage.xml"

  phpunit-pdo-oci:
    name: "PHPUnit on PDO_OCI"
    runs-on: "ubuntu-20.04"

    strategy:
      matrix:
        php-version:
          - "7.4"

    services:
      oracle:
        image: "wnameless/oracle-xe-11g-r2"
        ports:
          - "1521:1521"

    steps:
      - name: "Checkout"
        uses: "actions/checkout@v2"
        with:
          fetch-depth: 2

      - name: "Install PHP"
        uses: "shivammathur/setup-php@v2"
        with:
          php-version: "${{ matrix.php-version }}"
          extensions: "pdo_oci"
          coverage: "pcov"
          ini-values: "zend.assertions=1"

      - name: "Cache dependencies installed with composer"
        uses: "actions/cache@v2"
        with:
          path: "~/.composer/cache"
          key: "php-${{ matrix.php-version }}-composer-locked-${{ hashFiles('composer.lock') }}"
          restore-keys: "php-${{ matrix.php-version }}-composer-locked-"

      - name: "Install dependencies with composer"
        run: "composer install --no-interaction --no-progress --no-suggest"

      - name: "Run PHPUnit"
        run: "vendor/bin/phpunit -c ci/github/phpunit/pdo_oci.xml --coverage-clover=coverage.xml"

      - name: "Upload coverage file"
        uses: "actions/upload-artifact@v2"
        with:
          name: "${{ github.job }}-${{ matrix.php-version }}.coverage"
          path: "coverage.xml"

```

**Expected behavior**
No crash, extension installs cleanly

**Screenshots/Logs**
https://github.com/doctrine/dbal/pull/4355/checks?check_run_id=1270725004#step:4:16
https://github.com/doctrine/dbal/pull/4355/checks?check_run_id=1270725081#step:4:16

**Are you willing to submit a PR?**
I have no idea what is wrong here, so no.

**Comment by shivammathur** [comment]
@greg0ire Fixed in eaaab1c

**Comment by greg0ire** [comment]
It works again, thanks a lot!
